packages/gestalt/src/FormHelperTextCounter.js

```diff
@@ -0,0 +1,71 @@
+// @flow strict
+import { Fragment, useRef, useEffect, useState, type Node } from 'react';
+import Box from './Box.js';
+import Flex from './Flex.js';
+import Status from './Status.js';
+import Text from './Text.js';
+import { type MaxLength } from './TextField.js';
+
+type Props = {|
+  maxLength?: ?MaxLength,
```

Comment:
Since this component is gated by a truthy check on `maxLength` in FormHelperText, this type can be required, which will allow you to remove the optional chaining within this component

Reply:
DONE